tst/testall.g

```diff
@@ -1,31 +1,12 @@
-TestMyPackage := function( pkgname )
-local pkgdir, testfiles, testresult, ff, fn;
-LoadPackage( pkgname );
-pkgdir := DirectoriesPackageLibrary( pkgname, "tst" );
+#
+# This file runs package tests. It is also referenced in the package
+# metadata in PackageInfo.g.
+#
+LoadPackage( "OpenMath" );
 
-# Arrange testfiles as required
-testfiles := [ "openmath02.tst", "openmath03.tst", "test_new" ];
+TestDirectory(DirectoriesPackageLibrary( "OpenMath", "tst" ),
+  rec( exitGAP := true
+     , testOptions := rec(compareFunction := "uptowhitespace")));
```

Comment:
The comma on the new line... but if this is the only change in this PR, I can leave with this and fix this later.